src/main/resources/config/liquibase/changelog/20221117081200_changelog.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.4.xsd">
+    <!--
+    This changeset adds special grade name columns to grading scale for plagiarism and no participation grades.
+    -->
+    <changeSet author="akesfeden" id="20221117081200">
+        <addColumn tableName="grading_scale">
+            <column name="plagiarism_grade" type="VARCHAR(100)">
```

Comment:
size of 150 and 50 like here? https://github.com/ls1intum/Artemis/pull/5852/files#r1037968240

Reply:
Yes, I mean I intentionally tried 50 and 150 as values less than stated here to test the behavior. I could not observe a validation with `length`, but using `@Size` and `@Valid` blocks invalid length inputs with a `Http 400 Bad Request` error.